lib/index.js

```diff
@@ -106,6 +130,7 @@ exports.Model = class SchwiftyModel extends Model {
 
     parseValidationError(err) {
```

Comment:
It was based on the private function by the Objection guys: https://github.com/Vincit/objection.js/blob/b553097a58f8b5dac7ba78d1e9c72a5885f306d5/src/model/AjvValidator.js#L100

Since it's not even an override, I'll put a `_` even tho it seems like `$`'s are private in this world. Why would they use $?